src/libs/OptionsListUtils.ts

```diff
@@ -359,6 +359,12 @@ Onyx.connect({
     callback: (value) => (allReportsDraft = value),
 });
 
+let primaryPolicyID: OnyxEntry<string>;
```

Comment:
Can we call it activePolicyID as thats how the nvp key is called?

Reply:
@mountiny I updated.

Reply:
@nkdengineer @allgandalf I realize this is not ideal, but we already call the variables from the `NVP_ACTIVE_POLICY_ID` as activePolicyID elsewhere so I think we should just keep using that instead of using this term